docs/framework/data/adonet/ef/language-reference/how-to-call-custom-database-functions.md

```diff
@@ -18,7 +18,7 @@ The procedure below provides a high-level outline for calling a custom database
 
 1. Create a custom function in your database.
 
```

Comment:
Your script doesn't take into account if the URL has extra parameters and generates broken links in that case like the one below

Reply:
humm i think i found the problem!